README.md

```diff
@@ -1,18 +1,25 @@
-# Airbrake Plugin for Grails
+# Airbrake Plugin for Grails 3.1.4 > *
 
 This is the notifier plugin for integrating grails apps with [Airbrake](http://airbrake.io).
 
 When an uncaught exception occurs, Airbrake will POST the relevant data to the Airbrake server specified in your environment.
 
 ## Installation & Configuration
 
-Add the following to your `BuildConfig.groovy`
+Add the following to your `build.gradle`
 
 ```
-compile ":airbrake:0.9.4"
+repositories {
+    ...
+    maven { url "http://dl.bintray.com/marcatriflmedia/plugins" }
```

Comment:
I just packaged the plugin on my repo for testing purpose on my servers. It was easier than building the plugin on each of them. Of course it can be reversed. 
Again, i'm sorry if i made some mistakes here and there... I'm not used to github, it's the first time i use it so feel free to change whatever you want...